platform/state/platform_state.proto

```diff
@@ -81,6 +81,28 @@ message PlatformState {
      */
     proto.Timestamp last_frozen_time = 5;
 
+    /**
+     * A cryptographic hash of a roster.<br/>
+     * This hash represents a candidate roster, which is a roster that
+     * is not yet active, but is being considered.
+     * <p>
+     * A Node SHOULD NOT have more than one candidate roster at
+     * any given moment.<br/>
+     * This value MUST be a SHA-384 hash.
+     */
+    bytes candidate_roster_hash = 6;
+
+    /**
+     * A cryptographic hash of a roster.<br/>
+     * This hash represents an active roster, which is a roster that
+     * is adopted by the network and in current use.
+     * <p>
+     * The network SHALL NOT have more than one active roster at
+     * any given moment.<br/>
+     * This value MUST be a SHA-384 hash.
+     */
+    bytes active_roster_hash = 7;
+
```

Comment:
Does this belong to the PlatformState or a separate RosterState singleton per the design?

Also, I recall we wanted to replace the `active_roster_hash` with a list of round/roster_hash pairs.

Reply:
I paused updating this PR until fully ratified in Roster Proposal. Now that that's been accepted, I'll pull those updates here today.